packages/create-contentful-app/lib/index.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-console, no-process-exit */
+
+const chalk = require('chalk');
+const spawn = require('cross-spawn');
+const Spinner = require('cli-spinner').Spinner;
+const appFolder = process.argv[2];
+
+if (!appFolder) {
+  console.log();
+  console.log(
+    chalk.red(`Please provide a name for your app. E.g: "npx create-contentful-app my-app"`)
+  );
+  console.log();
+  process.exit(1);
+}
+
+function onSuccess(spinner, folder) {
+  spinner.stop(true);
+
+  console.log();
+  console.log(chalk.blue(`Success! Initialized "${folder}" in ~/${folder}.`));
```

Comment:
do we really need to repeat `folder` twice?

Reply:
@fabe wdyt about the wording here? Personally, "Initialized your contentful app in FOLDER_NAME" reads better.